src/tapis-app/Files/_components/Toolbar/Toolbar.tsx

```diff
@@ -35,14 +35,21 @@ export const ToolbarButton: React.FC<ToolbarButtonProps> = ({
 const Toolbar: React.FC<{ selectedFiles: Array<Files.FileInfo> }> = ({
   selectedFiles,
 }) => {
+  const [isOpen, setIsOpen] = useState(false)
```

Comment:
How will this work with other toolbars also trying to display modals? Perhaps the state should be a string that determines which modal is open. that way later we can do something like:

```
<NewFolderModal isOpen={ modal === 'newfolder' } />
```

Reply:
This also allows for one "closeModal" callback to be reused